src/workingtitle-vcockpits-instruments-navsystems/html_ui/Pages/VCockpit/Instruments/NavSystems/AS1000/MFD/AS1000_MFD.html

```diff
@@ -0,0 +1,968 @@
+                        <div>ETE</div>
+                        <div id="ETE"></div>
+                    </div>
+                    <div id="bottom"><span id="CurrentPageDisplay">MAP - NAVIGATION MAP</span></div>
+                </div>
+
+                <div id="Right">
+                    <div id="Com1">
+                        <div id="Com1_Active">--</div>
+                        <div id="Com1_DoubleArrow"><img src="/Pages/VCockpit/Instruments/NavSystems/Shared/Images/Misc/DoubleArrow.svg" /></div>
+                        <div class="Selectable" id="Com1_StdBy">--</div>
+                        <div>COM 1</div>
+                    </div>
+                    <div id="Com2">
+                        <div id="Com2_Active">--</div>
+                        <div id="Com2_DoubleArrow"><img src="/Pages/VCockpit/Instruments/NavSystems/Shared/Images/Misc/DoubleArrow.svg" /></div>
+                        <div class="Selectable" id="Com2_StdBy">--</div>
+                        <div>COM 2</div>
+                    </div>
+                </div>
+            </div>
+
+            <div id="SoftKeys">
+                <div id="Key1" class="SoftKey"></div>
+                <div id="Key2" class="SoftKey"></div>
+                <div id="Key3" class="SoftKey">MAP</div>
+                <div id="Key4" class="SoftKey"></div>
+                <div id="Key5" class="SoftKey"></div>
+                <div id="Key6" class="SoftKey"></div>
+                <div id="Key7" class="SoftKey"></div>
+                <div id="Key8" class="SoftKey"></div>
+                <div id="Key9" class="SoftKey"></div>
+                <div id="Key10" class="SoftKey">DCLTR</div>
+                <div id="Key11" class="SoftKey"></div>
+                <div id="Key12" class="SoftKey end"></div>
+            </div>
+
+            <div id="ContextualMenu" state="Inactive">
+                <div id="ContextualMenuContent">
+                    <div id="ContextualMenuTitle">EXAMPLE MENU</div>
+                    <div id="ContextualMenuMainArea">
+                        <div id="subTitle">OPTIONS</div>
+                        <div id="ContextualMenuElements">
+                            <div class="ContextualMenuElement" state="Inactive">Inactive?</div>
+                            <div class="ContextualMenuElement" state="Selected">Selected?</div>
+                            <div class="ContextualMenuElement" state="Unselected">Unselected?</div>
+                        </div>
+                        <div id="SliderMenu" state="Active">
+                            <div id="SliderMenuCursor"></div>
+                        </div>
+                    </div>
+                </div>
+                <div id="SubText">
+                    Press the FMS CRSR knob to return to base page
+                </div>
+            </div>
+        </div>
+    </div>
+</script>
+
+<link rel="stylesheet" href="AS1000_MFD.css" />
+<link rel="stylesheet" href="AS1000_MFD_Custom.css?53" />
```

Comment:
Let's knock that query string off of here.

Reply:
Sorted